eppo/src/main/java/cloud/eppo/android/ConfigurationStore.java

```diff
@@ -22,13 +22,15 @@
 import cloud.eppo.android.dto.FlagConfig;
 import cloud.eppo.android.dto.RandomizationConfigResponse;
 import cloud.eppo.android.dto.adapters.EppoValueAdapter;
+import cloud.eppo.android.dto.deserializers.RandomizationConfigResponseDeserializer;
 import cloud.eppo.android.util.Utils;
 
 public class ConfigurationStore {
 
     private static final String TAG = logTag(ConfigurationStore.class);
 
     private final Gson gson = new GsonBuilder()
+            .registerTypeAdapter(RandomizationConfigResponse.class, new RandomizationConfigResponseDeserializer())
```

Comment:
We now use a custom (vs. [automagic](https://github.com/google/gson/blob/main/UserGuide.md#json-field-naming-support)) deserializer.

This way, upstream users of our SDK who also use ProGuard modification don't need to modify their `proguard-rules.pro` file. (Note: This is what [LaunchDarkly requires](https://github.com/launchdarkly/android-client-sdk/blob/main/launchdarkly-android-client-sdk/consumer-proguard-rules.pro))

It will also protect against accidentally breaking things if we add new classes that for whatever reason are not covered by the rules, gson is updated, etc.